internal/reader/ccip.go

```diff
@@ -557,5 +602,33 @@ func (r *CCIPChainReader) validateWriterExistence(chains ...cciptypes.ChainSelec
 	return nil
 }
 
+// getSourceChainsConfig returns the destination offRamp contract's static chain configuration.
+func (r *CCIPChainReader) getOfframpStaticConfig(ctx context.Context) (offrampStaticChainConfig, error) {
+	if err := r.validateReaderExistence(r.destChain); err != nil {
+		return offrampStaticChainConfig{}, err
+	}
+
+	resp := offrampStaticChainConfig{}
+	err := r.contractReaders[r.destChain].GetLatestValue(
+		ctx,
+		consts.ContractNameOffRamp,
+		consts.MethodNameOfframpGetStaticConfig,
+		primitives.Unconfirmed,
+		map[string]any{},
+		&resp,
+	)
+	if err != nil {
+		return offrampStaticChainConfig{}, fmt.Errorf("failed to get source chain config: %w", err)
+	}
+	return resp, nil
+}
+
+type offrampStaticChainConfig struct {
```

Comment:
Can you put link to the offramp type? Will make it easier to find it.

Reply:
Yeah, I'll add a comment, what did you have in mind? A comment? I copied the pattern used for `type sourceChainConfig struct {`, so I'll update that as well.

Reply:
A permalink to the solidity struct would be good enough IMO